README.md

```diff
@@ -30,7 +30,7 @@ There are many flavors of time series data. Some can be windowed in the stream,
 
 
 ### Build the code 
-If this is your first time running SBT, you will be downloading the internet.
+If this is your first time running SBT, you will be downloading from the Internet.
```

Comment:
@hanmd82 "downloading the internet" is a phrase - it means: go get coffee, write a novel, come back and maybe it will be done by then. So this is correct as is :)